.hooks/bootstrap

```diff
@@ -118,7 +118,7 @@ else
                 exit 1
             fi
         fi
-        virtualenv -p $PY3CMD venv
+        virtualenv --system-site-packages -p $PY3CMD venv
```

Comment:
Can you please explain this change?

Reply:
Basically we wanted to get rid of this line in our code
`venv/bin/pip3 install -r .circleci/build-requirements.txt >> $ARTIFACTS_FOLDER/logs/installations.log 2>&1`
and here I did that by installing the build-requirements python packages inside the python installed on the image. 
If we use virtualenv, the python won't be able to access the system installed python packages, only the ones installed on the venv. This flag lets us use both: if the python package is not installed in the virtualenv the python interpreter will look for packages installed outside of it next. The quote from the virtualenv documentation is:
`Give the virtual environment access to the system site-packages dir.`
https://virtualenv.pypa.io/en/latest/cli_interface.html?highlight=site-packages#system-site-packages

That said I think I might need to add a flag also when installing the packages. TBC

Reply:
👍🏼